index.test.ts

```diff
@@ -0,0 +1,16 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import { CompatStatement } from './types/types.js';
+
+import bcd from './index.js';
+
+describe('Using BCD', () => {
+  it('subscript notation', () => {
+    const data: CompatStatement = bcd['api']['AbortController']['__compat'];
```

Comment:
> warning  'data' is assigned a value but never used

The easiest way to fix this is not to declare the variable:
```
  it('subscript notation', () => {
    bcd['api']['AbortController']['__compat'];
  });
```
Similarly on line 14.

Reply:
The purpose of the variable was to ensure that the types match.  I've added an `assert.ok()` call so this check can still be utilized!

Reply:
Ah, I see, yes that makes more sense!